game.py

```diff
@@ -7,8 +6,12 @@
 import os
 import random
 import math
+if os.environ.get('LOCAL_DEV'):
```

Comment:
agh my ruby brain wants me to extract this to a factory and have the decision injected in. what to do in python?! samesies?